orm/bucket.go

```diff
@@ -182,6 +185,8 @@ func (b Bucket) Save(db weave.KVStore, model Object) error {
 		return err
 	}
 
+	// TODO - ensure the metadata is set
```

Comment:
We also want to ensure latest version before Save (and error otherwise, or auto-migrate?)

Reply:
Of course, knowing what the latest version is, and how to migrate means importing `migrations`, which in turn imports `orm` for Bucket.

Let's just merge those two packages. Unless you have a better idea

Reply:
I am all for merging. Until I start working on the messages which require this functionality as well. [Transaction](https://github.com/iov-one/weave/pull/550#discussion_r278047980) might need it too. :see_no_evil: